src/System.IO.FileSystem/src/System/IO/Directory.Win32.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.IO
+{
+    public static partial class Directory
+    {
+        public static string[] GetLogicalDrives()
```

Comment:
Can we refactor out this logic into a helper shared with https://github.com/dotnet/corefx/blob/master/src/System.IO.FileSystem.DriveInfo/src/System/IO/DriveInfo.Windows.cs#L153 so that we're not duplicating the logic?